website/versioned_docs/version-5.x/ecosystem-css.md

```diff
@@ -0,0 +1,336 @@
+
+Each microfrontend should only load its CSS into the DOM after its javascript is downloaded. Single-spa lazy loads the javascript for each microfrontend, by default, so the CSS for the microfrontends will only be loaded as needed.
+
+## Unmounting CSS
+
+In large systems with dozens of microfrontends, it can become important for performance to unmount CSS as you navigate between pages. This is accomplished by removing `<style>` and `<link>` elements from the DOM.
+
+By default, most tooling will load and mount the CSS one time and leave it there indefinitely (it never unmounts!). However, some resources exist for unmounting CSS that is no longer being used, and remounting it once it's needed again.
+
+To accomplish this, single-spa applications and parcels should remove `<link>` and `<style>` elements inside of their [unmount lifecycle function](/docs/building-applications#unmount):
+
+```js
+// This code is an example of the mechanics of mounting + unmounting + remounting CSS.
+// In practice, this is often done via tools like css-loader, style-loader, or
+// single-spa-css (rather than manually).
+const style = document.createElement('style');
+style.textContent = `.settings {color: blue;}`;
+
+export const mount = [
+  async () => {
+    document.head.appendChild(styleElement);
+  },
+  reactLifecycles.mount,
+]
+
+export const unmount = [
+  reactLifecycles.unmount,
+  async () => {
+    styleElement.remove();
+  }
+]
+```
+
+To help you accomplish this, this [single-spa-css](/docs/ecosystem-css#single-spa-css) library implements mount and unmount functions for you.
+
+## SASS, PostCSS, Less, Stylus, etc
+
+[SASS](https://sass-lang.com/), [PostCSS](https://postcss.org/), [Less](https://lesscss.org/), [Stylus](https://stylus-lang.com/), and other CSS build tools are all compatible with single-spa and microfrontends.
+
+These tools run at build-time to produce vanilla CSS files. All of the documentation on this page applies to the output CSS files created by SASS and other CSS preprocessors.
+
+Since each microfrontend has its own build, this means that there are multiple SASS (or other preprocessor) builds occurring - one per microfrontend. As a result, SASS variables are not shareable via [cross microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports), since cross microfrontend imports occur at runtime. Instead, to share SASS variables, you'll need to publish them to an NPM registry and install them individually into each microfrontend. Since npm packages are not independently deployable (separately from the packages that use them), updates to the variables will need to be updated and deployed in each microfrontend individually. One thing to note is that [the browser's implementation of CSS variables](#css-variables) occurs at runtime and so native CSS modules are inherently shareable between microfrontends.
```

Comment:
```suggestion
Since each microfrontend has its own build, this means that there are multiple SASS (or other preprocessor) builds occurring - one per microfrontend. As a result, SASS variables are not shareable via [cross microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports), since cross microfrontend imports occur at runtime. Instead, to share SASS variables, you'll need to publish them to an NPM registry and install them individually into each microfrontend. Since npm packages are not independently deployable (separately from the packages that use them), changes to the variables will need to be updated and deployed in each microfrontend individually. One thing to note is that [the browser's implementation of CSS variables](#css-variables) occurs at runtime and so native CSS variables are inherently shareable between microfrontends.
```

Reply:
I think you meant variables here, but maybe I misunderstood what you're writing?

Reply:
You are correct, just a typo